### ISSUE TITLE
telegraf: Plugin fails to parse result with 1.3.7 release package

### ISSUE THREAD

**Issue by windkit** [leo-project/leofs#870]
When build from source, there is no issue.
```
$snmpwalk -v2c -c public 127.0.0.1:4010 .1.3.6.1.4.1.35450
iso.3.6.1.4.1.35450.34.1.0 = STRING: "test@test"
iso.3.6.1.4.1.35450.34.2.0 = Gauge32: 538
iso.3.6.1.4.1.35450.34.3.0 = Gauge32: 72424809
iso.3.6.1.4.1.35450.34.4.0 = Gauge32: 51886439
iso.3.6.1.4.1.35450.34.5.0 = Gauge32: 20537891
iso.3.6.1.4.1.35450.34.6.0 = Gauge32: 6355767
iso.3.6.1.4.1.35450.34.7.0 = Gauge32: 538
iso.3.6.1.4.1.35450.34.8.0 = Gauge32: 72504578
iso.3.6.1.4.1.35450.34.9.0 = Gauge32: 51881218
iso.3.6.1.4.1.35450.34.10.0 = Gauge32: 20609542
iso.3.6.1.4.1.35450.34.11.0 = Gauge32: 6356338
iso.3.6.1.4.1.35450.34.12.0 = Gauge32: 0
iso.3.6.1.4.1.35450.34.13.0 = Gauge32: 0
iso.3.6.1.4.1.35450.34.14.0 = Gauge32: 0
iso.3.6.1.4.1.35450.34.15.0 = Gauge32: 0
iso.3.6.1.4.1.35450.34.16.0 = Gauge32: 0
iso.3.6.1.4.1.35450.34.17.0 = Gauge32: 0
iso.3.6.1.4.1.35450.34.18.0 = Gauge32: 0
iso.3.6.1.4.1.35450.34.19.0 = Gauge32: 0
iso.3.6.1.4.1.35450.34.20.0 = Gauge32: 0
iso.3.6.1.4.1.35450.34.21.0 = Gauge32: 0
iso.3.6.1.4.1.35450.34.31.0 = Gauge32: 63
iso.3.6.1.4.1.35450.34.32.0 = Gauge32: 114881776
iso.3.6.1.4.1.35450.34.33.0 = Gauge32: 63
iso.3.6.1.4.1.35450.34.34.0 = Gauge32: 114106949
```

But the result from release package is different
```
$snmpwalk -v2c -c public 127.0.0.1:4010 .1.3.6.1.4.1.35450
SNMPv2-SMI::enterprises.35450.34.1.0 = STRING: "test@test"
SNMPv2-SMI::enterprises.35450.34.2.0 = Gauge32: 1251
SNMPv2-SMI::enterprises.35450.34.3.0 = Gauge32: 77323105
SNMPv2-SMI::enterprises.35450.34.4.0 = Gauge32: 40986185
SNMPv2-SMI::enterprises.35450.34.5.0 = Gauge32: 36165928
SNMPv2-SMI::enterprises.35450.34.6.0 = Gauge32: 7330953
SNMPv2-SMI::enterprises.35450.34.7.0 = Gauge32: 1251
SNMPv2-SMI::enterprises.35450.34.8.0 = Gauge32: 76225568
SNMPv2-SMI::enterprises.35450.34.9.0 = Gauge32: 40011071
SNMPv2-SMI::enterprises.35450.34.10.0 = Gauge32: 36096666
SNMPv2-SMI::enterprises.35450.34.11.0 = Gauge32: 7334446
SNMPv2-SMI::enterprises.35450.34.12.0 = Gauge32: 1775
SNMPv2-SMI::enterprises.35450.34.13.0 = Gauge32: 0
SNMPv2-SMI::enterprises.35450.34.14.0 = Gauge32: 0
SNMPv2-SMI::enterprises.35450.34.15.0 = Gauge32: 9346
SNMPv2-SMI::enterprises.35450.34.16.0 = Gauge32: 0
SNMPv2-SMI::enterprises.35450.34.17.0 = Gauge32: 0
SNMPv2-SMI::enterprises.35450.34.18.0 = Gauge32: 4088774
SNMPv2-SMI::enterprises.35450.34.19.0 = Gauge32: 4240986
SNMPv2-SMI::enterprises.35450.34.20.0 = Gauge32: 311482
SNMPv2-SMI::enterprises.35450.34.21.0 = Gauge32: 326732
SNMPv2-SMI::enterprises.35450.34.22.0 = Gauge32: 0
SNMPv2-SMI::enterprises.35450.34.23.0 = Gauge32: 0
SNMPv2-SMI::enterprises.35450.34.24.0 = Gauge32: 0
SNMPv2-SMI::enterprises.35450.34.31.0 = Gauge32: 55
SNMPv2-SMI::enterprises.35450.34.32.0 = Gauge32: 137061035
SNMPv2-SMI::enterprises.35450.34.33.0 = Gauge32: 54
SNMPv2-SMI::enterprises.35450.34.34.0 = Gauge32: 139119957

```

The parsing is expecting only one `:` as in https://github.com/influxdata/telegraf/blob/master/plugins/inputs/leofs/leofs.go#L240

**Comment by mocchira** [comment]
I think this should be dealt with on the telegraf plugin for leofs so please file the issue on influxdata/telegraf

**Comment by windkit** [comment]
Merged and will be included from 1.4.3
https://github.com/influxdata/telegraf/pull/3328

**Comment by mocchira** [comment]
@windkit Great.

**Comment by windkit** [comment]
Fixed in 1.4.3, released on 2017-10-25
